packages/hint/docs/about/PROJECT_CHARTER.md

```diff
@@ -0,0 +1,70 @@
+# webhint charter
+
+This project charter serves as a statement of scope and objectives for
+the **webhint** project.
+
+## 1. Guiding principles
+
+The **webhint** project is part of the [OpenJS Foundation], which
+operates transparently, openly, collaboratively, and ethically. Project
+proposals, timelines, and status must not merely be open, but also easily
+visible to outsiders.
+
+## 2. Scope
+
+Webhint's goal is to help web developers create better websites by
+providing feedback and best practices guidance in several areas such
+as (but not limited to) accessibility, security, performance, etc.
+
+Webhint should be useful during all the development lifecycle and
+adapt its feedback to each stage and the users needs: it can be run
+as a CLI, VS Code extension, browser extension and as an online
+service.
```

Comment:
```suggestion
adapt its feedback to each stage and the users' needs.
```

Reply:
Recommending this so that we don't need to update this as product suite based on webhint evolves.